examples/homepage/install/arm64/amplify.txt

```diff
@@ -17,5 +17,5 @@ General information
 - Name: <NAME-OF-FUNCTION>
 ? Which setting do you want to update? Lambda layers configuration
 ? Do you want to enable Lambda layers for this function? Yes
-? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:017000801446:layer:AWSLambdaPowertoolsPythonV2-Arm64:69
-? Do you want to edit the local lambda function now? No
\ No newline at end of file
+? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:017000801446:layer:AWSLambdaPowertoolsPythonV3-Arm64-312:69
```

Comment:
```suggestion
? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:017000801446:layer:AWSLambdaPowertoolsPythonV3-Arm64-312:1
```

question: version 1 like others?

Reply:
I had to update this with the new name format we decided on. It's up to date.